src/Administration/Resources/app/administration/src/module/sw-order/page/sw-order-list/sw-order-list.html.twig

```diff
@@ -69,7 +69,10 @@
                                 {% block sw_order_list_grid_columns %}
                                     {% block sw_order_list_grid_columns_customer_name %}
                                         <template #column-orderCustomer.firstName="{ item }">
-                                            {{ item.orderCustomer.firstName }} {{ item.orderCustomer.lastName }}
+                                            <router-link class="sw-data-grid__cell-value"
```

Comment:
Is this class needed because otherwise some styling is broken? Normally it should not be necessary to add this.  
```suggestion
                                            <router-link>
```

Reply:
I think you're right. I removed it. Works as expected without this class.

Reply:
Ok cool, I approved the PR 👍